ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -731,6 +736,29 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskLimbOption) {
                                               handOptions:(ORKPredefinedTaskHandOption)handOptions
                                                   options:(ORKPredefinedTaskOption)options;
 
+/**
+ Returns a predefined task that measures visual attention and task switching.
+ 
+ In a trail making test, the participant is asked to connect a series of cicles labelled 1,2,3... or
+ 1,A,2,B,3,C... and time to complete the test is recorded
+ 
+ @param identifier              The task identifier to use for this task, appropriate to the study.
+ @param intendedUseDescription  A localized string describing the intended use of the data
+                                  collected. If the value of this parameter is `nil`, the default
+                                  localized text is displayed.
+ @param trailmakingInstruction  Instructional content describing what the user needs to do when
+                                  the task begins. If the value of this parameter is `nil`,
+ @param trailType               Type of trail to display. Either @"A" or @"B"
```

Comment:
More descriptive type names would be preferable, or at least clarification of which is which.